controllers/providers/dns/infoblox.go

```diff
@@ -181,3 +165,27 @@ func (p *InfobloxProvider) SaveDNSEndpoint(gslb *k8gbv1beta1.Gslb, i *externaldn
 func (p *InfobloxProvider) String() string {
 	return "Infoblox"
 }
+
+func (p *InfobloxProvider) saveHeartbeatTXTRecord(objMgr *ibclient.ObjectManager, gslb *k8gbv1beta1.Gslb) (err error) {
+	var heartbeatTXTRecord *ibclient.RecordTXT
+	edgeTimestamp := fmt.Sprint(time.Now().UTC().Format("2006-01-02T15:04:05"))
+	heartbeatTXTName := fmt.Sprintf("%s-heartbeat-%s.%s", gslb.Name, p.config.ClusterGeoTag, p.config.EdgeDNSZone)
+	heartbeatTXTRecord, err = objMgr.GetTXTRecord(heartbeatTXTName)
+	if err != nil {
+		return
+	}
+	if heartbeatTXTRecord == nil {
+		log.Info().Msgf("Creating split brain TXT record(%s)...", heartbeatTXTName)
+		_, err = objMgr.CreateTXTRecord(heartbeatTXTName, edgeTimestamp, gslb.Spec.Strategy.DNSTtlSeconds, "default")
+		if err != nil {
+			return
+		}
+	} else {
+		log.Info().Msgf("Updating split brain TXT record(%s)...", heartbeatTXTName)
```

Comment:
Let's add `heartbeatTXTName` variable to the log message context
```suggestion
		log.Info().Str("HeartbeatTXTName", heartbeatTXTName).Msg("Updating split brain TXT record")
```